Add settings for Travis CI

giginet opened the pull request.

vincentisambart commented on README.md:
```diff
@@ -38,11 +41,19 @@ For information on how to use CocoaPods, please refer to the official CocoaPods
 Before you write your Pact definitions, you have to configure the mock server.
 
-Edit your test scheme and add the following to the `Pre-actions` section. (You might have to make changes to the PATH setting.)
+First, add `Gemfile` on project root
```
s/on/to

vincentisambart commented on README.md:
````diff
@@ -38,11 +41,19 @@ For information on how to use CocoaPods, please refer to the official CocoaPods
+gem "pact-mock_service"
+```
+
+Next, edit your test scheme and add the following to the `Pre-actions` section. (You might have to make changes to the PATH setting.)
````
s/Next/Then

vincentisambart commented on Tests/Phakchi/IntegrateTest.swift:
```diff
@@ -338,9 +338,15 @@ class IntegrateTestCase: XCTestCase {
+        let expectation = expectationWithDescription("session is closed")
+
+        // workaround to pass test on CI
+        let delayTime = dispatch_time(DISPATCH_TIME_NOW, Int64(1 * Double(NSEC_PER_SEC)))
```
ないとどういう問題が起きるんだろう

giginet replied:
実行タイミングによって転けたり通ったりします。
delay入れると転けなくなるようだ･･････

https://travis-ci.org/cookpad/Phakchi/jobs/159241184#L413

vincentisambart replied:
Shouldn't you just increase the timeout duration given to `waitForExpectationsWithTimeout`?

giginet commented:
#2 contains this changes, so I close this.

vincentisambart commented on README.md:
```diff
@@ -109,6 +120,17 @@ class RecipeClientPact: XCTestCase {
     }
+
+    override func tearDown() {
+        super.tearDown()
```
`super.tearDown()` should probably at the end of `tearDown()`.

giginet commented:
I removed such a bad workaround.
I modified timing to launch mock servers

https://github.com/cookpad/Phakchi/pull/2/commits/2c81920f7480a7b6fb4597e9fbacf8291f82bad1
